crates/binjs_io/src/io/statistics.rs

```diff
@@ -33,68 +33,63 @@ impl BytesAndInstances {
     }
 }
 
+/// A macro used to generate code that will operate on all fields of a `UserExtensible`.
 #[macro_export]
-macro_rules! for_field_in_content_info {
+macro_rules! for_field_in_user_extensible {
     ( $cb: ident ) => {
         $cb!(
-            (bools, "bools", b"bools"),
             (floats, "floats", b"floats"),
             (unsigned_longs, "unsigned_longs", b"unsigned_longs"),
-            (string_enums, "string_enums", b"string_enums"),
             (property_keys, "property_keys", b"property_keys"),
             (identifier_names, "identifier_names", b"identifier_names"),
-            (interface_names, "interface_names", b"interface_names"),
             (string_literals, "string_literals", b"string_literals"),
             (list_lengths, "list_lengths", b"list_lengths")
         )
     };
 }
 
-/// A container for information associated with a type of data we write to the stream
-/// as part of the content (i.e. not the header).
-///
-/// Typically used to collect/display the number of bytes written in each category.
+/// A container for information associated with user-extensible data (e.g.
+/// string literals).
```

Comment:
Can you define what is "user-extensible data" and what is "information associated with it" in more detail?

Also, the current struct name doesn't say "information associated with it" part, so this sounds like the data stored in the struct is "user-extensible data".